content/3_Silver/Sorting_Custom.mdx

```diff
@@ -640,6 +642,120 @@ public class Sol {
 
 </LanguageSection>
 
+## Coordinate Compression
+
+Coordinate compression is another useful technique related to sorting that can
+also be used in conjuction with custom comparators. Coordinate compression
+describes the process of replacing each value in a list with its index if that
+list was sorted. For example, the list $\{7, 3, 4, 1\}$ would be compressed to
+$\{3, 1, 2, 0\}$. Notice that $1$ is the least value in the first list, so it
+becomes $0$, and $7$ is the greatest value, so it becomes $3$, the largest index
+in the list.
+
+When we have values that can take on a large range of possible values, but we
+only care about their relative order (for example, if we have to know if one
+value is above another), coordinate compression is a simple way to help a lot
+with implementation. If we have a set of integers ranging from $0$ to $10^9$, we
+can't use them as array indices because we'd have to create an array of size
+$10^9$, which would surely cause a `Memory Limit Exceeded` verdict. However, if
+there are only $N = 10^6$ such integers, we can coordinate-compress their
+values, which guarantees that the values will all be in the range from $0$ to
+$N-1$, which _can_ be used as array indices.
+
+<FocusProblem problem="sample2" />
+
+A good example of coordinate compression in action is in the solution of _USACO
+Rectangular Pasture._ Again, we won't delve into the full solution but instead
+discuss how coordinate compression is applied. Since the solution uses
+[2D-prefix](/silver/prefix-sums-2) sums (another Silver topic), it is helpful if
```

Comment:
the only reason I don't particularly like this example is bc it doesn't handle cases where there doesn't have to be unique coordinates. Although this problem covers that, I'm sure there are others that don't. Not sure if the other reviewers have an opinion on this.